### ISSUE TITLE
Move to contrib?

### ISSUE THREAD

**Issue by garyb** [purescript-contrib/purescript-profunctor-lenses#6]
Would you consider moving this to contrib? We'll trash `photons` and just use this if it's somewhere where multiple maintainers can contribute/admin it.

**Comment by paf31** [comment]
I'm definitely happy to move it to contrib, but I'd like to use the usual issue/PR/review workflow.

**Comment by garyb** [comment]
For sure, it's more just for the _very_ unlikely event that you stop working on this/purescript :smile:

**Comment by paf31** [comment]
Well if that ever happens (as you say, unlikely), I'd definitely move my repos somewhere else, but they're all permissively licensed anyway, so using a fork in Bower should be doable.

**Comment by paf31** [comment]
The initial reason for this repo was as an experiment with `profunctors`, but it seems to have a dual purpose now as a more active `lens` replacement since @joneshf seems to be pretty busy these days. So I'd like for this to be a fairly stable API since things like Thermite and `optic-ui` depend on it. I might even switch over `derive-lenses` and `foreign-lenses` if people start using it.

**Comment by garyb** [comment]
Yeah, I remember that - I mentioned to Jon that I think it was intended to replace `lens` eventually when we talked about it the other day.

That's basically why we'd like to rely on it via contrib - so we don't end up in the `simple-dom`/`optics` situation we have now where there are minor changes that need making (version bumps and the like) but the original maintainer is busy with other things.

Forking is definitely an option when that happens, but it potentially means updating a whole bunch of other libraries as the dependency name will need changing for the fork too.

Stable is good, we don't really need anything other than what's already here aside from adding `Contains`, `Index`, `At`, which @jonsterling has put together and can PR on here, and then I added some prisms for `Coproducts`. Apart from that it's already awesome :)

**Comment by garyb** [comment]
Thanks!

**Comment by paf31** [comment]
Yep, I already saw the indexer stuff, on the `photons` repo. What I saw looked nice. Let's open a PR to just go over the module structure though.

And yes, this seems like a perfect candidate for contrib, since I'd like people to rely on it being around for a while. `lens` is technically pretty up to date, and compiles etc. but at least if it's in contrib, we can deprecate it or update it ourselves as need be.
